dev/breeze/src/airflow_breeze/commands/release_candidate_command.py

```diff
@@ -164,9 +164,9 @@ def push_artifacts_to_asf_repo(version, repo_root):
             check=True,
         )
         console_print("Files pushed to svn")
+        os.chdir(repo_root)
         # Remove old releases
         remove_old_releases(version, repo_root)
-        os.chdir(repo_root)
         run_command(["rm", "-rf", "asf-dist"], dry_run_override=DRY_RUN, check=True)
```

Comment:
I'm just seeing #30944, and I feel like instead of moving where we remove old releases, we shouldn't do this cleanup in the push method. My 2c.

Reply:
I think it would be packaged into the pypi package if we don't remove the asf-dist at this step. Do you prefer we check it out again when we want to check for the old releases?

Reply:
We create the asf-dist folder inside the repo

Reply:
How about we just split the push/cleanup/delete into discrete steps then?

Reply:
Fixed